docs/design/integration_nova.adoc

```diff
@@ -20,16 +20,194 @@ We would want to integrate with Nova to support the same set of user operations,
 
 == Review of OpenStack Workflow
 
+nova call neutron apis
+|===
+|Name |method |params |fields |neutron api |alcor status
+
+|show_network
+|get
+|net_id
+|provider:physical_network,provider:network_type
+|/v2.0/networks/{net_id}
+|/v4/{project_id}/vpcs/{vpcid}
+
+|show_network
```

Comment:
See multiple rows in this table with the same Name. Does this mean Nova calls Neutron or Alcor in multiple different scenarios, right? Can you specify the scenarios?